web/src/models/Transaction.model.ts

```diff
@@ -2,33 +2,43 @@ import {Model, TTransactionsSchemas} from 'types/Common.types';
 import Summary from './Summary.model';
 import Test from './Test.model';
 
+export type TRawTransactionResource = TTransactionsSchemas['TransactionResource'];
 export type TRawTransaction = TTransactionsSchemas['Transaction'];
 type Transaction = Model<
-  TRawTransaction,
+  TTransactionsSchemas['Transaction'],
   {
-    steps: Test[];
+    steps: string[];
+    fullSteps: Test[];
     summary: Summary;
   }
 >;
 
 function Transaction({
+  spec: rawTransaction = {},
+}: TRawTransactionResource): Transaction {
+  return Transaction.FromRawTransaction(rawTransaction);
+}
+
+Transaction.FromRawTransaction = ({
```

Comment:
Currently, we have to support two versions of the transactions API object:
- resource (from the resource endpoints)
- regular raw version (from the result list home page)